backend/models/bulletin.py

```
@@ -333,6 +470,39 @@ async def search_bulletins(
+		object_ids = [ObjectId(bid) for bid in bulletin_ids]
+	except Exception as exc:
+		print(f"Error: Invalid bulletin ID format in reorder: {exc}")
+		return False
+
+	try:
+		for idx, object_id in enumerate(object_ids):
+			await DB.db["bulletins"].update_one(
+				{"_id": object_id},
+				{"$set": {"order": idx, "updated_at": datetime.utcnow()}}
+			)
+		return True
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)